<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="4" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-01T20:54:30 PDT" name="Default suite" finished-at="2023-04-01T20:54:47 PDT" duration-ms="17546">
    <groups>
      <group name="chromeinvokation">
        <method signature="FieldPresenceValidations.F2301_AE_url_title(java.util.HashMap)[pri:1, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" name="F2301_AE_url_title" class="com.ae.shopping.testCases.FieldPresenceValidations"/>
      </group> <!-- chromeinvokation -->
    </groups>
    <test started-at="2023-04-01T20:54:30 PDT" name="Default test" finished-at="2023-04-01T20:54:47 PDT" duration-ms="17546">
      <class name="com.ae.shopping.testCases.FieldPresenceValidations">
        <test-method is-config="true" signature="launchApp()[pri:0, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:30 PDT" name="launchApp" finished-at="2023-04-01T20:54:43 PDT" duration-ms="13794" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method is-config="true" signature="makeItNull()[pri:0, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:44 PDT" name="makeItNull" finished-at="2023-04-01T20:54:44 PDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- makeItNull -->
        <test-method retried="true" signature="F2301_AE_url_title(java.util.HashMap)[pri:1, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:44 PDT" name="F2301_AE_url_title" data-provider="exptdDataSetUp" finished-at="2023-04-01T20:54:44 PDT" duration-ms="19" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{exptdUrlTitle=Men’s & Women’s Jeans, Clothes & Accessories | American Eagl, exptdOptFrmWomen=Tops, exptdProdName=AE Go Big Shirt}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Men’s & Women’s Jeans, Clothes & Accessories | American Eagl] but found [Men’s & Women’s Jeans, Clothes & Accessories | American Eagle]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Men’s & Women’s Jeans, Clothes & Accessories | American Eagl] but found [Men’s & Women’s Jeans, Clothes & Accessories | American Eagle]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.ae.shopping.testCases.FieldPresenceValidations.F2301_AE_url_title(FieldPresenceValidations.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- F2301_AE_url_title -->
        <test-method signature="F2301_AE_url_title(java.util.HashMap)[pri:1, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:44 PDT" name="F2301_AE_url_title" data-provider="exptdDataSetUp" finished-at="2023-04-01T20:54:44 PDT" duration-ms="15" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{exptdUrlTitle=Men’s & Women’s Jeans, Clothes & Accessories | American Eagl, exptdOptFrmWomen=Tops, exptdProdName=AE Go Big Shirt}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Men’s & Women’s Jeans, Clothes & Accessories | American Eagl] but found [Men’s & Women’s Jeans, Clothes & Accessories | American Eagle]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Men’s & Women’s Jeans, Clothes & Accessories | American Eagl] but found [Men’s & Women’s Jeans, Clothes & Accessories | American Eagle]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.ae.shopping.testCases.FieldPresenceValidations.F2301_AE_url_title(FieldPresenceValidations.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- F2301_AE_url_title -->
        <test-method is-config="true" signature="makeItNull()[pri:0, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:44 PDT" name="makeItNull" finished-at="2023-04-01T20:54:44 PDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- makeItNull -->
        <test-method is-config="true" signature="makeItNull()[pri:0, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:44 PDT" name="makeItNull" finished-at="2023-04-01T20:54:44 PDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- makeItNull -->
        <test-method signature="F2301_AE_womenMenu_topsOption_display(java.util.HashMap)[pri:2, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:44 PDT" name="F2301_AE_womenMenu_topsOption_display" data-provider="exptdDataSetUp" finished-at="2023-04-01T20:54:45 PDT" duration-ms="935" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{exptdUrlTitle=Men’s & Women’s Jeans, Clothes & Accessories | American Eagl, exptdOptFrmWomen=Tops, exptdProdName=AE Go Big Shirt}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- F2301_AE_womenMenu_topsOption_display -->
        <test-method signature="F2301_AE_product_display(java.util.HashMap)[pri:3, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:45 PDT" name="F2301_AE_product_display" data-provider="exptdDataSetUp" finished-at="2023-04-01T20:54:47 PDT" duration-ms="2485" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{exptdUrlTitle=Men’s & Women’s Jeans, Clothes & Accessories | American Eagl, exptdOptFrmWomen=Tops, exptdProdName=AE Go Big Shirt}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected product name from Now Trending header is not present expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected product name from Now Trending header is not present expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.ae.shopping.testCases.FieldPresenceValidations.F2301_AE_product_display(FieldPresenceValidations.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- F2301_AE_product_display -->
        <test-method is-config="true" signature="makeItNull()[pri:0, instance:com.ae.shopping.testCases.FieldPresenceValidations@534a5a98]" started-at="2023-04-01T20:54:45 PDT" name="makeItNull" finished-at="2023-04-01T20:54:45 PDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- makeItNull -->
      </class> <!-- com.ae.shopping.testCases.FieldPresenceValidations -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
